### ISSUE TITLE
Crash when deploying

### ISSUE THREAD

**Issue by jmahlman** [carlashley/appleLoops#34]
Installing loops for GarageBand and Logic Pro, command/script we use worked fine on 8/20/18 but stopped suddenly last night.

- Command used: 
`/usr/local/bin/appleloops --mute-progress-bar --deployment -m -o
`
- Version:
2.2.3

- Debug log output (last lines):

> 2018-08-24 10:42:03,769 - appleLoops - DEBUG - Loop(pkg_name=u'MAContent10_AssetPack_0069_AlchemySoundscapesMicroForest.pkg', pkg_url=u'http://audiocontentdownload.apple.com/lp10_ms3_content_2016/MAContent10_AssetPack_0069_AlchemySoundscapesMicroForest.pkg', pkg_mandatory=False, pkg_size=50302967, pkg_install_size=57822658, pkg_year='2016', pkg_loop_for='garageband', pkg_plist='garageband1020.plist', pkg_id=u'com.apple.pkg.MAContent10_AssetPack_0069_AlchemySoundscapesMicroForest', pkg_installed=False, pkg_destination=u'/tmp/MAContent10_AssetPack_0069_AlchemySoundscapesMicroForest.pkg', pkg_local_ver='0.0.0', pkg_remote_ver='0.0.0')
> 2018-08-24 10:42:03,770 - appleLoops - DEBUG - Traceback (most recent call last):
>   File "/usr/local/bin/appleloops", line 563, in main_processor
>     self.process_pkgs(self.get_feed(urls.apple, urls.fallback), os.path.basename(urls.apple))  # NOQA
>   File "/usr/local/bin/appleloops", line 928, in process_pkgs
>     update_pkg_sizes(_loop)
>   File "/usr/local/bin/appleloops", line 913, in update_pkg_sizes
>     self.size_info['download_total'] = self.size_info['download_total'] + loop.pkg_size  # NOQA
> TypeError: unsupported operand type(s) for +: 'int' and 'NoneType'
> 
> 2018-08-24 10:42:03,770 - appleLoops - DEBUG - Exception: unsupported operand type(s) for +: 'int' and 'NoneType'

**Comment by TSPARR** [comment]
Just chiming in that I'm seeing a very similar crash:

`sudo /usr/local/bin/appleLoops.py --dry-run --deployment -m -o
Password:
Dry run - loops download to: /tmp
Processing loops from: garageband1021.plist
Download: MAContent10_AssetPack_0539_DrummerTambourine.pkg (Package size: 19.75 MB  Install size: 34.63 MB)
  Install: MAContent10_AssetPack_0539_DrummerTambourine.pkg
Download: MAContent10_AssetPack_0634_AppleLoopsDrummerGraham.pkg (Package size: 4.12 MB  Install size: 4.35 MB)
  Install: MAContent10_AssetPack_0634_AppleLoopsDrummerGraham.pkg
.........
Download: MAContent10_AssetPack_0067_AlchemySoundscapesWater.pkg (Package size: 54.10 MB  Install size: 95.91 MB)
  Install: MAContent10_AssetPack_0067_AlchemySoundscapesWater.pkg
Download: MAContent10_AssetPack_0629_AppleLoopsDrummerAnders.pkg (Package size: 5.18 MB  Install size: 5.42 MB)
  Install: MAContent10_AssetPack_0629_AppleLoopsDrummerAnders.pkg
Traceback (most recent call last):
  File "/usr/local/bin/appleLoops.py", line 1601, in <module>
    main()
  File "/usr/local/bin/appleLoops.py", line 1593, in main
    al.main_processor()
  File "/usr/local/bin/appleLoops.py", line 572, in main_processor
    raise e
TypeError: unsupported operand type(s) for +: 'int' and 'NoneType'`

**Comment by TSPARR** [comment]
Just came back to report that this is now working for me. Something on Apple's end that was broke momentarily?

**Comment by jmahlman** [comment]
Lucky you, @TSPARR! 😛 

Still seeing the same issue here.

**Comment by jmahlman** [comment]
When trying to download the file at http://audiocontentdownload.apple.com/lp10_ms3_content_2016/MAContent10_AssetPack_0616_LogicAlchemyEssentials.pkg on our network it is marked as containing a Trojan: Virus/OSX.WGeneric.ritbo

Offsite I downloaded the file and can a shasum against the locally cached file we have from a few months ago as well as downloading from the HTTPS source and all sums match.  It looks like a false positive.  

I got the file whitelisted on our network.  I am closing this issue as it seems like it's not an issue with the script but with the network security.
